app/repository/family.py

```diff
@@ -50,33 +50,38 @@ def _get_query(db: Session) -> Query:
     )
 
 
-def _family_to_dto(db: Session, fam_geo_meta_org: FamilyGeoMetaOrg) -> FamilyReadDTO:
-    f = fam_geo_meta_org[0]
-    geo_value = cast(str, fam_geo_meta_org[1].value)
-    metadata = cast(dict, fam_geo_meta_org[2].value)
-    org = cast(str, fam_geo_meta_org[3].name)
+def _family_to_dto(
+    db: Session, fam_geo_meta_corp_org: FamilyGeoMetaOrg
+) -> FamilyReadDTO:
+    fam, geo, meta, corpus, org = fam_geo_meta_corp_org
+
+    geo_value = cast(str, geo.value)
+    metadata = cast(dict, meta.value)
+    org = cast(str, org.name)
     return FamilyReadDTO(
-        import_id=str(f.import_id),
-        title=str(f.title),
-        summary=str(f.description),
+        import_id=str(fam.import_id),
```

Comment:
We could go the whole hog and use `family` !? :laughing:

Reply:
Hahaha would normally agree but in the interest of keeping the unpacking easy to read and consistent for var names I'll stick with the somewhat shortened version for now!